frontend/amundsen_application/static/js/features/Widgets/SearchBar/InlineSearchResults/SearchItemList/tests/index.spec.tsx

```diff
@@ -13,7 +13,8 @@ import {
   indexUsersEnabled,
 } from 'config/config-utils';
 import SearchItemList, { SearchItemListProps } from '..';
-import SearchItem from '../SearchItem';
+// ignore eslint rule about unused imports
+import SearchItem from '../SearchItem'; // eslint-disable-line
```

Comment:
I think you are actually not using this at all. 
What you have in line 28 is called the same way, but it could be called in any other way.

Reply:
Yep you were right--I updated it.